config/style/positron.json

```diff
@@ -0,0 +1,1181 @@
+{
+  "version": 8,
+  "name": "Positron",
+  "sources": {
+    "openmaptiles": {
+      "type": "vector",
+      "url": "/v1/tiles/topographic/EPSG:3857/tile.json",
+      "attribution": "© 2022 Toitū Te Whenua © OpenMapTiles - CC BY 4.0"
+    }
+  },
+  "sprite": "https://openmaptiles.github.io/positron-gl-style/sprite",
+  "glyphs": "https://api.maptiler.com/fonts/{fontstack}/{range}.pbf?key={key}",
```

Comment:
we should our glyphs

Reply:
Do we also want to change the basic style with our own glyphs and sprites?